observability/network_observability/network-observability-operator-release-notes.adoc

```diff
@@ -13,6 +13,31 @@ These release notes track the development of the Network Observability Operator
 
 For an overview of the Network Observability Operator, see xref:../../observability/network_observability/network-observability-overview.adoc#dependency-network-observability[About Network Observability Operator].
 
+[id="network-observability-operator-release-notes-1-6-1_{context}"]
+== Network Observability Operator 1.6.1
+The following advisory is available for the Network Observability Operator 1.6.1:
+
+* link:https://access.redhat.com/errata/RHSA-2024:4785[2024:4785 Network Observability Operator 1.6.1]
+
+[id="network-observability-operator-CVE-1-6-1_{context}"]
+=== CVEs
+* link:https://access.redhat.com/errata/RHSA-2024:4237[RHSA-2024:4237]
+* link:https://access.redhat.com/errata/RHSA-2024:4212[RHSA-2024:4212]
+
+[id="network-observability-operator-1.6.1-bug-fixes_{context}"]
+=== Bug fixes
+* Previously, packet drops information, such as cause and TCP state, was only available in the Loki datastore and not in Prometheus. For that reason, the drop statistics in the OpenShift web console plugin *Overview* was only available with Loki. Now, packet drops information is also added to metrics, so you can view drops statistics when Loki is disabled. (link:https://issues.redhat.com/browse/NETOBSERV-1649[*NETOBSERV-1649*])
+* When the eBPF agent `PacketDrop` feature was enabled, and sampling was configured to a value greater than `1`, reported dropped bytes and dropped packets ignored the sampling configuration. While this was done on purpose, so as not to miss any drops, a side effect was that the reported proportion of drops versus non-drops became biased. For example, at a very high sampling rate, such as `1:1000`, it was likely that almost all the traffic appears to be dropped when observed from the console plugin. With this fix, the sampling configuration is honored with dropped bytes and packets. (link:https://issues.redhat.com/browse/NETOBSERV-1676[*NETOBSERV-1676*])
+* Previously, the SR-IOV secondary interface was not detected if the interface was created first and then the eBPF agent was deployed. It was only detected if the agent was deployed first and then the SR-IOV interface was created. Now, the SR-IOV secondary interface is detected no matter the sequence of the deployments. (link:https://issues.redhat.com/browse/NETOBSERV-1697[*NETOBSERV-1697*])
+* Previously, when Loki was disabled, the *Topology* view in the OpenShift web console displayed the *Cluster* and *Zone* aggregation options in the slider beside the network topology diagram, even when the related features are not enabled. This has been fixed, the slider now only display options according to the enabled features. (link:https://issues.redhat.com/browse/NETOBSERV-1705[*NETOBSERV-1705*])
```

Comment:
Should everything in the first sentence be in past tense?
```suggestion
* Previously, when Loki was disabled, the *Topology* view in the OpenShift web console displayed the *Cluster* and *Zone* aggregation options in the slider beside the network topology diagram, even when the related features were not enabled. This has been fixed, the slider now only display options according to the enabled features. (link:https://issues.redhat.com/browse/NETOBSERV-1705[*NETOBSERV-1705*])
```

Reply:
AH good catch yeah this was a known issue that became a bug fix, looks like holdover present tense.